app/models/campus_solutions/student_resources.rb

```diff
@@ -102,7 +102,7 @@ def filter_role_based_link_settings(link_settings)
         when :withdraw_from_semester_add
           true unless !roles[:law]
         when :higher_degrees_committee_form, :special_enrollment_petition
-          true unless (roles[:graduate] || roles[:law]) && !(is_jd_llm_only? || is_law_visiting?)
+          true unless ((roles[:graduate] || roles[:law]) && !(is_jd_llm_only? || is_law_visiting?))
```

Comment:
This was an accident that snuck in while troubleshooting another issue, I added a commit to remove this change.